lib/mix/lib/mix/scm/git.ex

```diff
@@ -94,8 +94,20 @@ defmodule Mix.SCM.Git do
     File.mkdir_p!(path)
 
     File.cd!(path, fn ->
-      git!(["init", "--quiet"])
-      git!(["--git-dir=.git", "remote", "add", "origin", opts[:git]])
+      git!(~w[-c core.hooksPath='' -c init.templateDir='' init --quiet])
```

Comment:
I don't think we need to pass `core.hooksPath` here.

Reply:
If it isn't needed, then we shouldn't keep it. I don't think consistency here matters. Different commands, different options.

Reply:
@hauleth and did you look into setting `--templates=""` instead here and below?